samples/contrib/aws-samples/ground_truth_pipeline_demo/mini-image-classification-pipeline.py

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env python3
 
+# Uncomment the apply(use_aws_secret()) below if you are not using OIDC
```

Comment:
See earlier comments. I believe adding these comments in every sample looks messy and/or confusing. Our README does a strong enough job about explaining how to authenticate properly enough that we shouldn't need to rely on them.